### DISCUSSION TITLE
Report Module missing Features

### DISCUSSION THREAD

**Discussion by FuXXz** [kimai/kimai#2509] [General]
Hello everybody,

i looked at kimai and think it's really, really good.
Due to the fact that it's so good, I'm all the more disappointed because some essential functions are missing.
The reports!
I've tried so many time records and almost all of them have basic functions in the reports.
Such as grouping with subtotals, saving reports.

Are there already wishes in this direction, or am I alone with this wish?

Greetings

**Comment by Solidus-Prime** [comment]
They aren't very in depth by default, but things like Crystal Reports can do a lot with the information. PHP is also pretty flexible if you know how to use it. You could probably find a freelancer to help you customize them.
